package.json

```diff
@@ -20,9 +20,9 @@
   "license": "MIT",
   "dependencies": {
     "@auth0/thumbprint": "0.0.6",
-    "ejs": "2.5.5",
+    "ejs": "^3.1.9",
     "jsonwebtoken": "^9.0.0",
-    "saml": "^1.0.0",
+    "saml": "^3.0.1",
```

Comment:
Reasons for major bump:
1 -> 2: Require Node >= 12.
2 -> 3: Handle poorly formatted PEM files.